models/review.go

```diff
@@ -684,6 +687,10 @@ func DeleteReview(r *Review) error {
 		return fmt.Errorf("review is not allowed to be 0")
 	}
 
+	if r.Type == ReviewTypeRequest {
+		return fmt.Errorf("Can't delet review request by this way")
```

Comment:
typo

Reply:
I'm sorry. But I can't fully understand your suggestion. I'm not good at English. can you give me more detail. Thanks

Reply:
Please replace "delet" with "delete".